Cargo.toml

```diff
@@ -23,5 +23,5 @@ lto = "thin"
 codegen-units = 32
 
 [workspace.dependencies]
-tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "a8b400f" }
-tantivy-common = { git = "https://github.com/paradedb/tantivy.git", rev = "0c3318a" }
+tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "febfba4" }
```

Comment:
I pushed your rebase to the `main` branch of Tantivy. I think we should just specify the `main` branch here rather than a specific commit